tests/python/contrib/test_cmsisnn/utils.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""CMSIS-NN functions for testing networks"""
+
+import platform
+
+import numpy as np
+import pytest
+
+import tvm
+from tvm import relay
+
+
+def skip_if_no_reference_system(func):
```

Comment:
should we improve this a bit and make it less brittle? it seems like if we ever use this from a different test (e.g. TOPI tests run on GPU) we may need to broaden it a bit. i'm not sure Corstone300 provides a good way to detect it from TVM--but perhaps you guys know better? we could just add a pytest mark and flag `--enable-corstone300-tests` and add the flag to the relevant tests/scripts/task_*.sh file.

Reply:
cc @manupa-arm